src/pages/permissions/PermissionGroups.tsx

```diff
@@ -0,0 +1,298 @@
+import {
+  Button,
+  EmptyState,
+  Icon,
+  Row,
+  TablePagination,
+  useNotify,
+} from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import Loader from "components/Loader";
+import ScrollableTable from "components/ScrollableTable";
+import SelectableMainTable from "components/SelectableMainTable";
+import SelectedTableNotification from "components/SelectedTableNotification";
+import { FC, useState } from "react";
+import { queryKeys } from "util/queryKeys";
+import useSortTableData from "util/useSortTableData";
+import { getIdentitiesForGroup } from "util/permissionIdentities";
+import usePanelParams, { panels } from "util/usePanelParams";
+import CustomLayout from "components/CustomLayout";
+import PageHeader from "components/PageHeader";
+import NotificationRow from "components/NotificationRow";
+import HelpLink from "components/HelpLink";
+import { useDocs } from "context/useDocs";
+import { fetchGroups } from "api/auth-groups";
+import GroupActions from "./actions/GroupActions";
+import CreateGroupPanel from "./panels/CreateGroupPanel";
+import EditGroupPanel from "./panels/EditGroupPanel";
+import PermissionGroupsFilter from "./PermissionGroupsFilter";
+import EditGroupIdentitiesBtn from "./actions/EditGroupIdentitiesBtn";
+import EditGroupIdentitiesPanel from "./panels/EditGroupIdentitiesPanel";
+import { pluralize } from "util/instanceBulkActions";
+import BulkDeleteGroupsBtn from "./actions/BulkDeleteGroupsBtn";
+import EditGroupPermissionsPanel from "./panels/EditGroupPermissionsPanel";
+
+const PermissionGroups: FC = () => {
+  const notify = useNotify();
+  const {
+    data: groups = [],
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [queryKeys.authGroups],
+    queryFn: () => fetchGroups(),
+  });
+  const docBaseLink = useDocs();
+  const panelParams = usePanelParams();
+  const [search, setSearch] = useState("");
+  const [selectedGroupNames, setSelectedGroupNames] = useState<string[]>([]);
+
+  if (error) {
+    notify.failure("Loading groups failed", error);
+  }
+
+  const headers = [
+    { content: "Name", className: "name", sortKey: "name" },
+    { content: "Description", sortKey: "description" },
+    {
+      content: "Identities",
+      sortKey: "identities",
+      className: "u-align--right",
+    },
+    {
+      content: "Permissions",
+      sortKey: "permissions",
+      className: "u-align--right",
+    },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const filteredGroups = groups.filter(
+    (group) =>
+      !search ||
+      group.name.includes(search) ||
+      group.description.includes(search),
```

Comment:
Better do lowercase comparision?

Reply:
right, updated search in `EditIdentityGroupsPanel` and `EditGroupPermissionsPanel` as well for lowercase comparison.